apps/calendar/js/templates/multi_day.js

```diff
@@ -69,7 +69,7 @@ multiday.event = function() {
     </i>`;
   }
 
-  var className = ['md__event'];
+  var className = ['md__event', 'md__event-like-button'];
```

Comment:
Super small nit: unfortunately facebook has polluted my frame of reference for "like button", so I keep needing to do another pass on this name. I think you could get by without the "like" part of the name. But really up to you, not a big deal.